package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "eslint \"packages/**/*.ts\" \"**/__tests__/**/*.ts\"",
     "lint:packages": "eslint \"packages/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
     "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
-    "update:imperative": "npm i --save --save-exact @zowe/imperative@latest && syncpack fix-mismatches --dev --prod --filter @zowe/imperative",
+    "update:imperative": "lerna clean -y && npm i --save --save-exact @zowe/imperative@latest && syncpack fix-mismatches --dev --prod --filter @zowe/imperative && npm i",
```

Comment:
On my machine it seems like `npm i --save --save-exact @zowe/imperative@latest` causes Imperative to get duped 😢 

Perhaps we want to try this instead?
`"update:imperative": "npm i --package-lock-only --no-package-lock --save-exact @zowe/imperative@latest && syncpack fix-mismatches --dev --prod --filter @zowe/imperative && npm i",`

Reply:
thank you! 😋